src/components/HelpIcon.tsx

```diff
@@ -9,7 +9,13 @@ type HelpIconProps = {
 };
 
 export const HelpIcon = (props: HelpIconProps) => (
-  <label title={`${props.title} — ?`} className="help-icon">
-    <div onClick={props.onClick}>{questionCircle}</div>
-  </label>
+  <button
+    className="help-icon"
+    onClick={props.onClick}
+    type="button"
+    title={`${props.title} — ?`}
+    aria-label={props.title}
+  >
+    <div aria-hidden="true">{questionCircle}</div>
```

Comment:
```suggestion
    {questionCircle}
```
Do we need this extra div ?

Reply:
Since the SVG doesn't have any label, I'm using the `div` to pass `aria-hidden="true"` similar to what's being done in `ToolButton.tsx`.

Reply:
the svg itself has `aria-hidden` true so then this shouldn't be needed. We might have missed this in `ToolButton.tsx`.